docs/pull_request_template.md

```diff
@@ -0,0 +1,63 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+You can refer to [Kubernete's community documentation] on writing good commit messages, which provides good tips and ideas.
+
+Some PRs addess specific issues. Please, refer to the [CONTRIBUTING] documentation for more
+information on how to link a PR to an existing issue.
+
+# What environments does this code impact?
+
+- [ ] Cloud
+- [ ] Onprem
+- [ ] Controller
+- [ ] Operator
+- [x] None
+
+
+# How was this code tested?
+
+Please, select one or more if needed:
+
+- [ ] Local unit-tests
+- [ ] Local Kupe API subsystem tests
```

Comment:
Misspelled Kube API?

Reply:
Nope, I have changed it forever!

Reply:
Kute API :)

Reply:
Also on this line, but unrelated to spelling is that we have subsystem tests that are not for the kube-api.
I don't think those are covered by any of these bullets (unless you include that in `unit-tests`, which I wouldn't expect).

Maybe just drop the `Kube/Kube API` here entirely and leave it as just subsystem?

Reply:
> Maybe just drop the Kupe/Kube API here entirely and leave it as just subsystem?

:+1: